src/components/CharacterCardList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, ReactNode } from 'react';
+import { Character } from '../types';
+
+interface CharacterCardListProps {
+  characters: Character[];
+  isLoading: boolean;
+}
+
+class CharacterCardList extends Component<CharacterCardListProps> {
+  render(): ReactNode {
+    const { characters, isLoading } = this.props;
+    return (
+      <>
+        {isLoading ? (
+          <div className="d-flex align-items-center justify-content-around my-3">
+            <strong>Loading...</strong>
+            <div className="spinner-border ml-auto" role="status" aria-hidden="true"></div>
+          </div>
+        ) : null}
```

Comment:
{isLoading && (
          <div className="d-flex align-items-center justify-content-around my-3">
            <strong>Loading...</strong>
            <div className="spinner-border ml-auto" role="status" aria-hidden="true"></div>
          </div>
        )}
        
        it is a better way to make the same. And the best would be to move all this to a seperate coponent - f.e., Loader and just use 
        {isLoading &&  <Loader />